### PR TITLE
[HFRONT-696] fix: datepicker

### PR DESCRIPTION
**Pull request by snqb**
now it doesn't fail on the borders, doesn't disappear or jump around
just plain works

### CODE REVIEW

File: src/components/datepicker/index.tsx [hunk 1]
```diff
@@ -1,74 +1,84 @@
...
   return (
-    <Flex flexDirection="column" sx={{ ...styles(datePickerAlign), ...sx }}>
+    <Flex flexDirection="column" sx={{ ...styles, ...sx }}>
+      <ReactDatePicker
+        onCalendarOpen={() => setOpen(true)}
+        onCalendarClose={() => setOpen(false)}
+        shouldCloseOnSelect
+        {...props}
+        customInput={<DateSelect isOpen={isOpen} selectProps={selectProps} />}
+        popperPlacement={getPlacement(datePickerAlign)}
```
Comment (snqb) [inline src/components/datepicker/index.tsx:36]: mostly I just used `popper` api to fix the issue

**Review by vatj — APPROVED** [review]
Tested on VM, both validation drawer and activity "double" date pickers

**Review by esonhey — APPROVED** [review]
Great job!

**Pull request merged**